src/main/java/org/apache/parquet/format/InterningProtocol.java

```diff
@@ -224,6 +224,11 @@ public ByteBuffer readBinary() throws TException {
     return delegate.readBinary();
   }
 
+  @Override
+  public int getMinSerializedSize(byte b) throws TException {
+    return 0;
```

Comment:
I think, we should return `delegate.getMinSerializedSize(b)` instead of `0`.

Reply:
Good one, thanks!